config/templates/cloud-config-worker

```diff
@@ -144,27 +144,32 @@ coreos:
     {{ if .Experimental.NodeDrainer.Enabled }}
     - name: kube-node-drainer.service
       enable: true
+      command: start
+      runtime: true
       content: |
         [Unit]
         Description=drain this k8s node to make running pods time to gracefully shut down before stopping kubelet
-        Wants=kubelet.service docker.service
+        After=multi-user.target
 
         [Service]
         Type=oneshot
         RemainAfterExit=true
-        ExecStartPre=/usr/bin/systemctl is-active kubelet.service
-        ExecStartPre=/usr/bin/systemctl is-active docker.service
         ExecStart=/bin/true
-        ExecStop=/bin/sh -c '/usr/bin/docker run --rm -v /etc/kubernetes:/etc/kubernetes {{.HyperkubeImageRepo}}:{{.K8sVer}} \
-          /hyperkube kubectl \
+        TimeoutStopSec=30s
+        ExecStop=/bin/sh -c '/usr/bin/rkt run \
+        --volume=kube,kind=host,source=/etc/kubernetes,readOnly=true \
+        --mount=volume=kube,target=/etc/kubernetes \
+        --net=host \
+        quay.io/coreos/hyperkube:v1.4.5_coreos.0 \
+          --exec=/kubectl -- \
           --server=https://{{.ExternalDNSName}}:443 \
           --kubeconfig=/etc/kubernetes/worker-kubeconfig.yaml \
-          drain $$(hostname) \
+          drain $(hostname) \
```

Comment:
I now understand that `$$` and `$` to be equivalent in this context because it isn't recognized/expanded as systemd variables anyways. Good!

ref https://github.com/coreos/kube-aws/issues/40#issuecomment-258863442
ref https://github.com/coreos/kube-aws/pull/41#issuecomment-259003781